settings.py

```diff
@@ -18,6 +18,7 @@
 
 ICU_DATA_FOLDER_UNIX = "/usr/share/stpyv8"
 ICU_DATA_FOLDER_OSX  = "/Library/Application Support/STPyV8/"
+ICU_DATA_FOLDER_WINDOWS = os.path.join(os.environ["PROGRAMDATA"], "STPyV8")
```

Comment:
This setting breaks the build workflows on non Windows systems. I suggest replacing it with the following line

`ICU_DATA_FOLDER_WINDOWS = os.path.join(os.environ.get("PROGRAMDATA", ""), "STPyV8")`

Reply:
Better to set the whole variable to `None` if that envar doesn't exist. Fixing now